webpack.js

```diff
@@ -0,0 +1,74 @@
+const path = require('path');
+const webpack = require('webpack');
+const fs = require('fs');
+const filePath = path.join(__dirname, 'dist/index.js');
+const outputPath = path.join(__dirname, 'build.json');
+const appIconPath = path.join(__dirname, 'appIcon.png');
+
+const base64_encode = file => {
+  // read binary data
+  const bitmap = fs.readFileSync(file);
+  // convert binary data to base64 encoded string
+  return new Buffer(bitmap).toString('base64');
+};
+
+const obj = JSON.parse(
+  fs.readFileSync(path.join(__dirname, 'dapp_config.json'), 'utf8')
+);
+
+const jsonData = obj ? { ...obj, image: base64_encode(appIconPath) } : {};
+
```

Comment:
same here. If write file would only be about to write a file were content and the file name is passed in it would be fine. However, it's writing the build file. Which is something else.